be/src/fs/fs.h

```diff
@@ -290,6 +290,9 @@ class WritableFile {
     // implementation source code (e.g., env_posix.cc)
     virtual Status appendv(const Slice* data, size_t cnt) = 0;
 
+    // Compatible with native writev which has return written bytes size
+    virtual Status appendv(const Slice* data, size_t cnt, size_t* bytes_written) = 0;
+
```

Comment:
`appendv` will fail on partial write, so the result value of `*bytes_written` will always equals to the accumulated size of `data`, I don't think it's necessary to add this new API.